packages/webapp/src/util/siteMapConstants.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ *  Copyright 2023 LiteFarm.org
```

Comment:
2024! 😆 
Thank you for working on this, I will start testing now!

Reply:
I'm surprised we didn't have more of that these last two weeks!